src/main/resources/db/migration/0200/V246__ModulesDeliverables.sql

```diff
@@ -0,0 +1,108 @@
+CREATE TABLE accelerator.deliverable_categories (
+    id INTEGER PRIMARY KEY,
+    name TEXT NOT NULL UNIQUE
+);
+
+CREATE TABLE accelerator.deliverable_statuses (
+    id INTEGER PRIMARY KEY,
+    name TEXT NOT NULL UNIQUE
+);
+
+CREATE TABLE accelerator.deliverable_types (
+    id INTEGER PRIMARY KEY,
+    name TEXT NOT NULL UNIQUE
+);
+
+CREATE TABLE accelerator.document_stores (
+    id INTEGER PRIMARY KEY,
+    name TEXT NOT NULL UNIQUE
+);
+
+CREATE TABLE accelerator.modules (
+    id BIGINT PRIMARY KEY GENERATED BY DEFAULT AS IDENTITY,
+    name TEXT NOT NULL,
+    created_by BIGINT NOT NULL REFERENCES users,
+    created_time TIMESTAMP WITH TIME ZONE NOT NULL,
+    modified_by BIGINT NOT NULL REFERENCES users,
+    modified_time TIMESTAMP WITH TIME ZONE NOT NULL
+);
+
+CREATE TABLE accelerator.cohort_modules (
+    id BIGINT PRIMARY KEY GENERATED BY DEFAULT AS IDENTITY,
+    cohort_id BIGINT NOT NULL REFERENCES accelerator.cohorts,
+    module_id BIGINT NOT NULL REFERENCES accelerator.modules,
+    start_date DATE NOT NULL,
+    end_date DATE NOT NULL
+);
+
+CREATE INDEX ON accelerator.cohort_modules (cohort_id, start_date);
+
+CREATE TABLE accelerator.deliverables (
+    id BIGINT PRIMARY KEY GENERATED BY DEFAULT AS IDENTITY,
+    deliverable_category_id INTEGER NOT NULL REFERENCES accelerator.deliverable_categories,
+    deliverable_type_id INTEGER NOT NULL REFERENCES accelerator.deliverable_types,
+    module_id BIGINT NOT NULL REFERENCES accelerator.modules,
+    position INTEGER NOT NULL,
+    created_by BIGINT NOT NULL REFERENCES users,
+    created_time TIMESTAMP WITH TIME ZONE NOT NULL,
+    modified_by BIGINT NOT NULL REFERENCES users,
+    modified_time TIMESTAMP WITH TIME ZONE NOT NULL,
+    name TEXT NOT NULL,
+    subtitle TEXT,
+    description_html TEXT,
+    is_sensitive BOOLEAN NOT NULL,
+    is_required BOOLEAN NOT NULL,
+
+    UNIQUE (id, deliverable_type_id),
+    UNIQUE (module_id, position)
+);
+
+CREATE TABLE accelerator.deliverable_documents (
+    deliverable_id BIGINT NOT NULL REFERENCES accelerator.deliverables,
+    deliverable_type_id INTEGER NOT NULL REFERENCES accelerator.deliverable_types,
```

Comment:
Will this be the same value as in the deliverable (referenced by deliverable_id)? Need this here mostly for the constraint check?

Reply:
Yes, this is for the constraint check.